lte/gateway/c/sctpd/src/sctpd.cpp

```diff
@@ -75,11 +83,35 @@ int signalHandler(int* end, std::unique_ptr<Server>& server,
   return 0;
 }
 
+static magma::mconfig::SctpD load_sctpd_mconfig() {
+  magma::mconfig::SctpD mconfig;
+  if (!magma::load_service_mconfig_from_file(SCTPD_SERVICE, &mconfig)) {
+    mconfig.set_log_level(magma::orc8r::LogLevel::INFO);
```

Comment:
can we instead try to get it first from sctpd.yml file before hardwiring the default here?

Reply:
in terms of priority yml file takes precedence over streamed down config which takes precedence over hardwired log level as the convention in AGW.

Reply:
There is no sctpd.yml config file yet. Should we introduce one for the log level?

Reply:
Added a second commit with a yaml config.